L1Trigger/L1TMuonBarrel/src/L1TMuonBarrelParamsHelper.cc

```diff
@@ -36,7 +36,7 @@ void L1TMuonBarrelParamsHelper::configFromPy(std::map<std::string, int>& allInts
 	setAssLUTPath(AssLUTpath);
 	///Read Pt assignment Luts
 	std::vector<LUT> pta_lut(0); pta_lut.reserve(19);
-	std::vector<int> pta_threshold(6); pta_threshold.reserve(9);
+	std::vector<int> pta_threshold(6); pta_threshold.reserve(10);
```

Comment:
@ Yes, declaring vector initially with the correct size should also work.  If preferred can make this change. Please advise.